app/services/power_manager/base.rb

```diff
@@ -58,9 +56,11 @@ def translate_status(result)
     attr_reader :host
 
     def find_action_entry(method)
-      action_entry = action_map[method.to_sym] || {} # create a valid entry
+      action_entry = action_map[method.to_sym] || method # create a valid entry
       # if action_map is in simple format (:key => 'method'), transform it to entry
-      action_entry = {:action => action_entry.to_sym} unless action_entry.is_a?(Hash)
+      action_entry = {:default => action_entry.to_sym} unless action_entry.is_a?(Hash)
+      print "#{method} #{action_entry} \n"
```

Comment:
we should not do this ^_^ ... probably a debug leftover, but if you really want to write to the logs, use [logger.debug](https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels)

Reply:
yep debug left over 😊